packages/monaco/src/browser/monaco-editor-service.ts

```diff
@@ -17,13 +17,14 @@
 import { injectable, inject, decorate } from 'inversify';
 import URI from '@theia/core/lib/common/uri';
 import { OpenerService, open, WidgetOpenMode, ApplicationShell, PreferenceService } from '@theia/core/lib/browser';
-import { EditorWidget, EditorOpenerOptions, EditorManager } from '@theia/editor/lib/browser';
+import { EditorWidget, EditorOpenerOptions, EditorManager, CustomEditorWidget } from '@theia/editor/lib/browser';
 import { MonacoEditor } from './monaco-editor';
 import { MonacoToProtocolConverter } from './monaco-to-protocol-converter';
 
 import ICodeEditor = monaco.editor.ICodeEditor;
 import CommonCodeEditor = monaco.editor.CommonCodeEditor;
 import IResourceInput = monaco.editor.IResourceInput;
+import { MonacoEditorModel } from './monaco-editor-model';
```

Comment:
minor: move the import statement with the others:

```ts
import { MonacoEditor } from './monaco-editor';
import { MonacoEditorModel } from './monaco-editor-model';
```

Reply:
Fixed